src/plugins/discover/public/services/doc_views/components/doc_viewer_table/table.tsx

```diff
@@ -108,6 +109,8 @@ export const DocViewerTable = ({
   onAddColumn,
   onRemoveColumn,
 }: DocViewRenderProps) => {
+  const showActionsInsideTableCell = useIsWithinBreakpoints(['xl'], true);
```

Comment:
OH! TIL, this is a handy hook, FYI @davismcphee who's also working on responsive stuff currently! 👍

Reply:
Yup this is really convenient to have, I'm using it in my layout PR too.